src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-form/iscsi-target-form.component.html

```diff
@@ -0,0 +1,549 @@
+            <div class="panel panel-default"
+                 *ngFor="let group of groups.controls; let i = index"
+                 [formGroupName]="i">
+              <div class="panel-heading">
+                <ng-container i18n>Group</ng-container>: {{ group.getValue('group_id') }}
+                <button type="button"
+                        class="close"
+                        (click)="groups.removeAt(i)">
+                  <i class="fa fa-remove fa-fw"></i>
+                </button>
+              </div>
+              <div class="panel-body">
+                <!-- Group: group_id -->
+                <div class="form-group">
+                  <label class="control-label col-sm-3"
+                         for="group_id">
+                    <ng-container i18n>Name</ng-container>
+                    <span class="required"></span>
+                  </label>
+                  <div class="col-sm-9">
+                    <input class="form-control"
+                           type="text"
+                           formControlName="group_id">
+                  </div>
+                </div>
+
+                <!-- Group: members -->
+                <div class="form-group"
+                     [ngClass]="{'has-error': group.showError('members', formDir)}">
+                  <label class="control-label col-sm-3"
+                         for="members">
+                    <ng-container i18n>Initiators</ng-container>
+                  </label>
+                  <div class="col-sm-9">
+                    <ng-container *ngFor="let member of group.getValue('members'); let i = index">
+                      <div class="input-group cd-mb">
+                        <input class="form-control"
+                               type="text"
+                               [value]="member"
+                               disabled />
+                        <span class="input-group-btn">
+                          <button class="btn btn-default"
+                                  type="button"
+                                  (click)="removeGroupInitiator(group, i)">
+                            <i class="fa fa-remove fa-fw"
+                               aria-hidden="true"></i>
+                          </button>
+                        </span>
+                      </div>
+                    </ng-container>
+
+                    <div class="row">
+                      <div class="col-md-12">
+                        <cd-select [data]="group.getValue('members')"
+                                   [options]="groupMembersSelections"
+                                   [messages]="messages.groupInitiator"
+                                   (selection)="onGroupMemberSelection($event)"
+                                   elemClass="btn btn-default pull-right">
+                          <i class="fa fa-fw fa-plus"></i>
+                          <ng-container i18n>Add Initiator</ng-container>
```

Comment:
Use ``Add Initiator`` or ``Add initiator``, but use the same typing across the whole code. Please adapt other text that uses ``Add|Edit|Create|... xxx``.

Reply:
Done, all are lowercase now.